lib/maths/CPrior.cc

```diff
@@ -304,6 +305,19 @@ std::string CPrior::debug() const {
     return std::string();
 }
 
+CPrior::TStrStrPr CPrior::printMarginalLikelihoodStatistics() const {
+    TStrStrPr unknownValuePr{UNKNOWN_VALUE_STRING, UNKNOWN_VALUE_STRING};
+
+    if (this->isNonInformative()) {
+        return unknownValuePr;
```

Comment:
You can construct a pair from an intializer list, so this line and the other one that returns `unknownValuePr` could be `return {UNKNOWN_VALUE_STRING, UNKNOWN_VALUE_STRING};`.  Then you wouldn't have to construct `unknownValuePr` in the happy day case of a useful mean and standard deviation being available.